src/components/policy/policy_external/include/policy/policy_manager_impl.h

```diff
@@ -1096,6 +1102,11 @@ class PolicyManagerImpl : public PolicyManager {
   bool IsPTValid(std::shared_ptr<policy_table::Table> policy_table,
                  policy_table::PolicyTableType type) const;
 
+  /**
+   * @brief Check that application for PTU more than zero
```

Comment:
@ydementieiev
```suggestion
   * @brief Check that new applications for PTU were registered
```

Reply:
@AKalinich-Luxoft done in [fe7b393](https://github.com/smartdevicelink/sdl_core/pull/3208/commits/fe7b393fe80f89ac9137d6dd9b41a0dfd6ba28c6)